src/ScrollToTop.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router';
+
```

Comment:
You'll want to put `import PropTypes from 'prop-types';` here because the way that ReactRouter described the fix by implementing the `React.PropTypes...` chain is actually deprecated now!